data/README.md

```diff
@@ -25,17 +27,24 @@ data
 ├── matterport3d
 │   ├── <scene_id>
 │   ├── ...
+├── arkitscenes
+│   ├── Training
+│   |   ├── <scene_id>
+│   |   ├── ...
+│   ├── Validation
+│   |   ├── <scene_id>
+│   |   ├── ...
 ├── embodiedscan_occupancy
 ├── embodiedscan_infos_train.pkl
 ├── embodiedscan_infos_val.pkl
 ├── embodiedscan_infos_test.pkl
-├── embodiedscan_infos_train_vg.json
-├── embodiedscan_infos_val_vg.json
-├── embodiedscan_infos_test_vg.json
-├── embodiedscan_infos_train_mini_vg.json (mini set)
-├── embodiedscan_infos_val_mini_vg.json (mini set)
-├── embodiedscan_infos_train_vg_all.json (w/ complex prompts)
-├── embodiedscan_infos_val_vg_all.json (w/ complex prompts)
+├── embodiedscan_train_vg.json
```

Comment:
change these names is not necessary?

Reply:
For consistency with released filenames. See [#71](https://github.com/OpenRobotLab/EmbodiedScan/issues/71#issuecomment-2334208369)